jina/serve/runtimes/gateway/grpc/gateway.py

```diff
@@ -37,13 +40,25 @@ def __init__(
         self.ssl_certfile = ssl_certfile
         self.health_servicer = health.HealthServicer(experimental_non_blocking=True)
 
+    def grpc_extra_server_interceptors(self) -> Optional[List['ServerInterceptor']]:
```

Comment:
why do I need this? Please remove

Reply:
@JoanFM This is added for custom gateway development. When the user wants to develop a gateway with more custom interceptors, he only needs to override this method (as shown in the example in the description of this PR). Without this method, the user must overwrite the `async def setup_server(self)`. I don't think this is a good approach.

Reply:
This decision will be made in the scope of the CustomGateway design, we will decide which methods are offered and which not. For now, user needs to override `setup_server` so please do so.

Reply:
well, I reverted this change.